pkg/data/components/OTelHTTPExporter.yaml

```diff
@@ -39,21 +48,15 @@ properties:
     type: int
     validations: [int]
     default: 443
+    advanced: true
   - name: Insecure
     summary: Provide a way to disable TLS export.
     description: |
       Can be used to send data without TLS.
     type: bool
     validations: [bool]
     default: false
-  - name: Headers
-    summary: Headers to emit when sending HTTP traffic.
-    description: |
-      Sending data to a backend may require additional headers to be
-      configured. This properties supports sending a map of header keys and
-      values.
-    type: map
-    validations: []
+    advanced: true
```

Comment:
Later we'll want to think about whether insecure is advanced, or if we should make any automatic decisions or add additional context. We found that insecure true was necessary when sending to refinery, or another collector, but insecure false is necessary for sending directly to honeycomb.

Reply:
"Send this to Refinery" is a separate component -- that's the TraceConverter.